Assets/Scripts/Model/AbstractPlayer.cs

```diff
@@ -52,6 +52,7 @@ protected virtual void Kill()
         {
             if (remainingLives == 0)
             {
+                GameManager.instance.OnEnemyKilled();
```

Comment:
Dlaczego to w ogóle jest w AbstractPlayer. I ten kod co jest wyżej w OnExplode ma if'a. Nie lepiej to zrobić w subklasach Player i Monster?